src/pages/GoogleLogin.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+const GoogleLogin = () => {
+  const history = useHistory();
+  useEffect(() => {
+    googleLogin();
+  }, []);
+
+  const googleLogin = () => {
+    window.gapi.load('auth2', function () {
+      // Retrieve the singleton for the GoogleAuth library and set up the client.
+      window.auth2 = window.gapi.auth2.init({
+        client_id:
+          '348690319815-t5e8gq77l8f3iqm60aqsiebna9utntq8.apps.googleusercontent.com',
+        cookiepolicy: 'single_host_origin',
+        //scope: 'additional_scope'
+      });
+      attachSignin(document.getElementById('customBtn'));
```

Comment:
React에서 dom 요소에 접근할 때 Ref를 써보시는건 어떠신지리산? ⛰
```suggestion
const customBtnRef = React.createRef();

attachSignin(customBtnRef);

<GoogleButton ref={customBtnRef}></GoogleButton>
```

Reply:
useRef 좋네요! 리뷰 국국~~~~~~👍🏻👍🏻👍🏻👍🏻